book/src/rfcs/ffadd.md

```diff
@@ -0,0 +1,192 @@
+# RFC: Foreign Field Addition
+
+This document is meant to explain the foreign field addition gate in Kimchi.
+
+## Overview
+
+The goal of this gate is to perform the following operation between a left $a$ and a right $b$ input, to obtain a result $r$
+$$a + b = r \mod f$$
+where $a,b,r\in\mathbb{F}_f$ belong to the foreign field of modulus $f$ and we work over a native field $\mathbb{F}_n$ with modulus $n$. 
+
+If $f<n$ then we can easily perform the above computation. But in this gate we are interested in the contrary case in which $f>n$. In order to deal with this, we will divide foreign field elements into limbs that fit in our native field. We want to be compatible with the foreign field multiplication gate, and thus the parameters we will be using are the following:
+
+- 3 limbs of 88 bits each (for a total of 264 bits)
+- $2^{264} > f$
+- our foreign field will have 256-bit length
```

Comment:
It may be better to state the bounds that work with our limb configuration instead of these specifics.